README.rst

```diff
@@ -37,6 +37,10 @@ automatic differentiation of hybrid quantum-classical computations.
 `Strawberry Fields <https://strawberryfields.readthedocs.io>`__ is a full-stack Python library
 for designing, simulating, and optimizing photonic quantum circuits.
 
+.. warning::
+    This plugin is no longer maintained and is not compatible with the latest version of PennyLane.
+    It is supported for version 0.29 of PennyLane and below.
+
```

Comment:
Thanks @lillian542! Please could we also have this callout in the docs page, i.e., [here](https://docs.pennylane.ai/projects/strawberryfields/en/latest/) just below the release.

Reply:
The first 3 lines of that page are pulled from this file, putting it here should put it both places:

```
:Release: |release|

.. include:: ../README.rst
  :start-after:	header-start-inclusion-marker-do-not-remove
  :end-before: header-end-inclusion-marker-do-not-remove
```

Reply:
Not at the very top though, I can rearrange the order

Reply:
https://xanaduai-pennylane--134.com.readthedocs.build/projects/strawberryfields/en/134/